### PR TITLE
feat(backup): re-add export selection to text file

### LINKED ISSUE #300: feat(backup): re-add export selection to text file
Per "issue" title. This feature in previous versions was very useful for me to keep track of what I had/had not debloated on different phones. I imported the export files into a spreadsheet column for each phone.
Many thanks.

### PR DESCRIPTION
**Pull request by edgy-b**
- added back possibility to export current selection to txt file

 - on successful export show modal

Resolves #300

### CODE REVIEW

File: src/gui/views/list.rs [hunk 1]
```diff
@@ -545,6 +580,40 @@ impl List {
...
+                Space::new(Length::Fill, Length::Shrink),
+            ];
+
+            let ctn = container(column![title, text_box, file_row, modal_btn_row])
+                .height(Length::Shrink)
+                .width(500)
+                .padding(10)
+                .style(style::Container::Frame);
+
+            return Modal::new(content.padding(10), ctn)
```
Comment (lavafroth) [inline src/gui/views/list.rs:612]: We might not need the explicit return here. Just a nit though.

File: src/gui/views/list.rs [hunk 2]
```diff
@@ -116,6 +119,7 @@ impl List {
             Message::ModalHide => {
                 self.selection_modal = false;
```
Comment (lavafroth) [inline src/gui/views/list.rs:120]: Now that we have so many modals, we might be better off with creating some sort of enum like `self.focused_modal = Modal::Selection` and then `match` upon that. Anyways, you don't have to implement it here. Let's do that in a separate commit / PR.

**Review by lavafroth — APPROVED** [review]
Looks good otherwise. Maybe add some documentations to keep deepsource happy.

**Comment by edgy-b** [comment]
> Looks good otherwise. Maybe add some documentations to keep deepsource happy.

You're right. Will do that in core/DRY branch, since I already started on cleaning to tidying code.
